 surfaceflinger: use min realtime sched priority

Use the rlimit service option to allow surfaceflinger to use the
minimum realtime scheduler priority. Surfaceflinger uses this priority
to minimize latency of vsync events.

diff --git a/services/surfaceflinger/surfaceflinger.rc b/services/surfaceflinger/surfaceflinger.rc
index 4e122da..064e100 100644
--- a/services/surfaceflinger/surfaceflinger.rc
+++ b/services/surfaceflinger/surfaceflinger.rc
@@ -2,5 +2,7 @@ service surfaceflinger /system/bin/surfaceflinger
     class core
     user system
     group graphics drmrpc readproc
+    # allow the minimum realtime scheduler priority: RLIMIT_RTPRIO(14)=1
+    rlimit 14 1 1
     onrestart restart zygote
     writepid /dev/stune/foreground/tasks
